src/index.js

```diff
@@ -483,7 +483,7 @@ class Collection {
 
 	/**
 	 * Set a value in the collection by key
-	 * @param {string} key - The key of the element you want to edit
+	 * @param {string|number} key - The key of the element you want to edit
```

Comment:
```suggestion
	 * @param {string | number} key - The key of the element you want to edit
```
small formatting tweak

Reply:
That's not really relevant